### DISCUSSION TITLE
DDNS and ddclient

### DISCUSSION THREAD

**Discussion by martinbutt** [openmiko/openmiko#77] [Ideas]
Has there been any thought put towards how best to add DDNS to Openmiko? Perhaps a new submodule of the [ddclient](https://github.com/ddclient/ddclient) library?

**Comment by beankylla** [comment]
wouldn't ddclient best be on some other device? seems like a bit off for a camera firmware no?
like a pi 0 or smth.

**Comment by martinbutt** [comment]
Can you explain more about your proposed architecture?

**Comment by beankylla** [comment]
i mean setting another device on the network that would manage this (router, raspberry pi, etc.)
